packages/apps/transfer/src/components/Common/Layout/partials/Footer/Footer.tsx

```diff
@@ -0,0 +1,100 @@
+import { Footer, SystemIcon, useModal } from '@kadena/react-ui';
+
+import { linkClass } from './styles.css';
+
+import { OptionsModal } from '@/components/Global/OptionsModal';
+import Link from 'next/link';
+import { useTheme } from 'next-themes';
+import useTranslation from 'next-translate/useTranslation';
+import React, { FC } from 'react';
+
+const FooterWrapper: FC = () => {
+  const { theme, setTheme } = useTheme();
+  const { renderModal } = useModal();
+
+  const { t } = useTranslation('common');
+  const openModal = (): void => renderModal(<OptionsModal />);
+
+  const toggleTheme = (): void => {
+    const newTheme = theme === 'dark' ? 'light' : 'dark';
```

Comment:
Can you base the default theme on the user's system settings? https://stackoverflow.com/a/57795518/5816268

Reply:
I believe the `next-themes` library already does that by default according to [this](https://www.npmjs.com/package/next-themes). I also checked the app local storage and when not defined, it is set to system
![image](https://github.com/kadena-community/kadena.js/assets/112877112/764f3cca-f1c5-4563-b5ad-a2b8dcfc0cdd)